buildstockbatch/workflow_generator/residential_hpxml.py

```diff
@@ -82,7 +82,16 @@ def create_osw(self, sim_id, building_id, upgrade_idx):
 
         logger.debug('Generating OSW, sim_id={}'.format(sim_id))
 
-        sim_ctl_args = {}
+        sim_ctl_args = {
+            'simulation_control_timestep': 60,
+            'simulation_control_run_period_begin_month': 1,
+            'simulation_control_run_period_begin_day_of_month': 1,
+            'simulation_control_run_period_end_month': 12,
+            'simulation_control_run_period_end_day_of_month': 31,
+            'simulation_control_run_period_calendar_year': 2007,
+            'debug': False,
+            'add_component_loads': False
+        }
```

Comment:
This seems out of scope for this PR.

Reply:
It _is_, but it _isn't_.

It prevents a warning that the hash is empty, and I'm doing it in this pending PR anyway: https://github.com/NREL/buildstockbatch/pull/274.